TestBackupRestoreIntegration intermediate failure

Test ending:
```
608713e0f469","dc":"","node_id":"","task_id":"00000000-0000-0000-0000-000000000000","keyspace":null,"snapshot_tag":"","min_date":"0001-01-01T00:00:00Z","max_date":"0001-01-01T00:00:00Z","temporary":false}}
--- And: add more data
--- Then: there is 15 rows
--- When: delete data
--- And: restore snapshot
    service_backup_integration_test.go:2116: stdout 
    service_backup_integration_test.go:2117: stderr chown: changing ownership of '/var/lib/scylla/data/system/truncated-38c19fd0fb863310a4b70d0cc66628aa/me-52-big-TOC.txt.tmp': No such file or directory
        chown: changing ownership of '/var/lib/scylla/data/system/truncated-38c19fd0fb863310a4b70d0cc66628aa/me-52-big-Digest.crc32': No such file or directory
        
    service_backup_integration_test.go:2118: Command failed on host 2001:db9:200:0:0:0:0:11 Process exited with status 1
--- FAIL: TestBackupRestoreIntegration (12.09s)
```

[Whole test logs](https://github.com/scylladb/scylla-manager/files/12148024/TestBackupRestoreIntegration.log)

--- a/pkg/service/backup/service_backup_integration_test.go
+++ b/pkg/service/backup/service_backup_integration_test.go
@@ -2111,7 +2111,7 @@ func TestBackupRestoreIntegration(t *testing.T) {
 	}
 
 	for _, nis := range status {
-		stdout, stderr, err := ExecOnHost(nis.Addr, "chown scylla:scylla -R /var/lib/scylla/data")
+		stdout, stderr, err := ExecOnHost(nis.Addr, "chown scylla:scylla -R /var/lib/scylla/data/"+testKeyspace)
 		if err != nil {
 			t.Log("stdout", stdout)
 			t.Log("stderr", stderr)

